wp-content/civi-extensions/civirazorpay/api/v3/Civirazorpay/CivicrmRazorpayFetchSettlementDateCron.php

```
@@ -0,0 +1,355 @@
+          // Skip if already settled.
+          if (!empty($contribution['Contribution_Details.Settlement_Id'])) {
+            $returnValues['processed']++;
+            \Civi::log()->debug('Contribution already settled', [
+              'contribution_id' => $contribution['id'],
+              'payment_id' => $paymentId,
+              'date' => $date,
+            ]);
+            continue;
+          }
+
+          $updateResult = Contribution::update(FALSE)
```

PHPMD (2.15.0)

Avoid unused local variables such as '$updateResult'. (Unused Code Rules)

(UnusedLocalVariable)